jobs/periodic/flag-old-conversations/index.ts

```diff
@@ -0,0 +1,94 @@
+import moment from 'moment';
+import { getMatch, getSubcourse } from '../../../graphql/util';
+import { getAllConversations, markConversationAsReadOnly, sendSystemMessage } from '../../../common/chat';
+import { getLogger } from '../../../common/logger/logger';
+import { Lecture } from '../../../graphql/generated';
+import { SystemMessage } from '../../../common/chat/types';
+import systemMessages from '../../../common/chat/localization';
+
+const logger = getLogger();
+
+enum ConversationType {
+    GROUP = 'group',
+    ONE_ON_ONE = 'one_on_one',
+}
+
+type conversationsToDeactivate = {
+    id: string;
+    conversationType: ConversationType;
+};
+async function isActiveMatch(id: number): Promise<boolean> {
+    const today = moment().endOf('day');
+    const match = await getMatch(id);
+    const dissolvedAtPlus30Days = moment(match.dissolvedAt).add(30, 'days');
+    return !dissolvedAtPlus30Days.isBefore(today);
```

Comment:
I'm probably missing some insights, but why is a match "active" for 30 days after it was dissolved?

Reply:
perhaps the naming is misleading. The point is that a one to one chat that came about because of a match whose match was dissolved 30 days ago is set as readonly. So the chat users have 30 days left to chat together.

Reply:
What do you think about adding this information as a comment to the function?